docs/ibexa_products/experience.md

```diff
@@ -0,0 +1,132 @@
+
+#### Site Factory
+
+[Site Factory ](site_factory.md) is a site management interface, integrated with the Back Office. It enables you to configure new sites without leaving the administration interface and editing SiteAccess configuration.
+With this feature you can create and deploy multiple websites at lightning speed and at scale.
+It allows you to manage expenses and resources while industrializing your web presence. Additionally, together with localized information and tailored product catalogs and prices, it helps you to quickly enter new markets.
+
+#### Customizable Dashboard
+
+Starting from Experience edition of [[= product_name =]] you can [customize the Dashboard]([[= user_doc =]]/getting_started/dashboard/work_with_dashboard/#customize-dashboard).
+You can create customized Dashboard using Dashboard Builder. Dashboard then can be tailored to your specific needs - it contains a set of widgets selected by the user. You can easily preview the sections that you use more often and omit the less significant ones.
+
+![Customizable dashboard](customizable_dashboard.png)
+
+#### Publish Later
+
+You can take complete control of where and when your content blocks are visible to your predefined audiences and [schedule content publication]([[= user_doc =]]/content_management/schedule_publishing/).
+[[= product_name_exp =]] comes with a Publish Later feature that allows you to schedule personalized content and reach different user groups at optimal dates and times to boost performance.
+What's more, you can turn specific content pages and blocks on and off to meet the needs of your marketing campaigns and promotions.
+Publish Later feature combined with Page Builder allows you to show all changes that you plan to the page for the future - just use the slider to see all the upcoming changes.
+
+#### Customer Portal
+
+Through the [[= product_name =]] Customer Portal and customer management that comes with it, you can manage company accounts, establish new ones, and communicate with your partners in a personalized space.
+With the help of this feature, you can create customized areas that give users a smooth, integrated experience and provide them access to a variety of resources, apps, and services from a single point of entry.
+Using this tool, your customers can change their organization details, invite and see members, self-register, and more.
+
+#### Segments
+
+[Segmentation](https://doc.ibexa.co/en/latest/administration/admin_panel/segments_admin_panel/) allows you to split up the user base.
+Using segments you can display specific content to specific Users and tailor the content that the visitors can see.
+The Targeting and Dynamic targeting blocks available in the Page Builder use them right out of the box.
+Segments work also with the [Personalization](personalization_brochure.md), where you can assign users to different recommendation groups and create advanced logic with operators to get best segmentation for your audience.
```

Comment:
```suggestion
The Targeting block is available in the Page Builder and you can use it right out of the box.
Segments work also with the [Personalization](personalization_brochure.md), where you can assign users to different recommendation groups and create advanced logic with operators to devlier the best recommendations for your audience.
```
1) There are some words missing in the first sentence
1) The Dynamic targeting block requires the Personalization integration (which is mentioned a sentence later)
2) Personalization delivers recommendations and I'd focus on that- the segmentation has to be done before that (manually or with CDP)